### ISSUE TITLE
Migrate dataset - DOI: 10.5061/dryad.cc2fqz64z	

### ISSUE THREAD

**Issue by Ariana-Orona** [CDL-Dryad/dryad-product-roadmap#1588]
Author is unable to view dataset under 'My Datasets' although they have not logged in with an alternate email. Can you please check on this and let me know if the author needs to log in with another address/if the dataset needs to be migrated?

Isabel Wagner
ORCID ID: 0000-0001-8764-149X
Email: isabelviola.wagner@uksh.de

Dataset information:

Title: Diabetes type 1 negatively influences Leydig cell function in rats which is partially reversible by insulin treatment
DOI: 10.5061/dryad.cc2fqz64z

**Comment by sfisher** [comment]
Hi @Ariana-Orona I looked at this dataset and user and I don't see anything unusual about it.  Her user record has her correct ORCID so she should be able to see it under her datasets after she logs in with the  ORCID shown in this ticket.

I added her email address to her user record since ORCID often doesn't give us an email and defaults to not giving it to us.

I tested moving it to my own account and it appears OK to me and then moved it back to her account afterward and it looks fine.

![Screen Shot 2022-01-04 at 12 25 51 PM](https://user-images.githubusercontent.com/105433/148119960-a2ac481b-e643-46e3-9c85-18c4a1b84e20.png)

From everything I can tell, she should be able to see this under the submitted datasets under "My Datasets" unless she is logging into ORCID as a different user.  I also notice we have someone with her name under ORCID 0000-0002-4287-0544, so maybe she is creating more extra orcid accounts and logging in with them.  It kind of seems like maybe she has 4 ORCID accounts if all the Isabel Wagners are her: https://orcid.org/orcid-search/search?searchQuery=isabel%20wagner

It kind of defeats the purpose of ORCID by having multiple accounts, but rather they recommend updating email addresses and information with them rather than creating multiple accounts for the same person.

Let me know if I should just move this to ORCID 0000-0002-4287-0544 instead.
